references/cli/debug.go

```diff
@@ -380,11 +380,12 @@ func (d *debugOpts) separateBySteps(v cue.Value, ioStreams cmdutil.IOStreams) er
 }
 
 type renderOptions struct {
-	hideIndex    bool
-	filterFields []string
+	// hideIndex    bool
```

Comment:
commented code

Reply:
Yeah sorry I want to update the `debug` tool at last, first to make all tests pass.

Reply:
:) Looking forward to merging this once the tests are passing